config/config.go

```diff
@@ -228,6 +228,14 @@ type ScrapeConfig struct {
 	Scheme string `yaml:"scheme,omitempty"`
 	// The HTTP basic authentication credentials for the targets.
 	BasicAuth *BasicAuth `yaml:"basic_auth,omitempty"`
+	// The bearer token for the targets.
+	BearerToken string `yaml:"bearer_token,omitempty"`
```

Comment:
@brian-brazil Doesn't this mask it? https://github.com/prometheus/prometheus/pull/921/files#diff-3baf47c64847a8fb8aaa8cc2e088513bR23